citar-file.el

```diff
@@ -82,6 +82,14 @@ separator that does not otherwise occur in citation keys."
                  (const :tag "Find files with space after key" "[[:space:]]")
                  (regexp :tag "Filename separator")))
 
+(defcustom citar-file-open-external-extensions '("html")
+  "List of file extensions to be opened with external application.
+
+These are the extensions that `citar-file-open' will open via
+`citar-file-open-external'."
+  :group 'citar
+  :type '(repeat string))
```

Comment:
The other approach is suggested in #507, where this might instead be:

```
(defcustom citar-file-open-extensions (list ("html" . 'citar-open-external)
                                            (t . 'find-file)))
```

Reply:
Yeah, I agree that it's more future-proof to make this an alist as you suggest. @ShuguangSun, are you willing to implement this? Otherwise I can take a look, but probably not right away.

Reply:
Just checking back, @ShuguangSun; can you answer @roshanshariff's question?

> ... are you willing to implement this?

E.g. make it match the plans in #507?

Otherwise, we may need to address this post-1.0 (which, hopefully, we can tag soon, pending a couple of minor things).

Reply:
I'll work on it.